### PR TITLE
Create 'legend' for API reference tags

### PR DESCRIPTION
**Pull request by jacobprall**
# Description

Creates a page for explanations regarding the API reference tags. Particularly useful for the Experimental and Experimental-toolkit tags, as the Community tag will direct users to the license comparison page and the Toolkit tag will point users to the install toolkit page.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

### CODE REVIEW

**Comment by ryanbooz** [comment]
Love the new license explanation. It's interesting having it separate from the API page. Hmmmmm....

Overall, I like the basic outline of the questions at the top, but I'm not a fan of how it gets rendered. I wonder if this might be more clear in a table with a question on the left, answer on the right (makes it easy to scan, too).

Initial impressions, but overall I love that we'll finally have this to point to!

File: api/api-tag-overview.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# API Reference Tag Overview
+
+Timescale Documentation utilizes a tagging system to describe functions in the API Reference section. These tags include "Community", "Experimental", "Toolkit", and "Experimental" (for TimescaleDB Toolkit functions). Below is more information on what each tag represents.
```
Comment (Loquacity) [inline api/api-tag-overview.md:3]:
```suggestion
The Timescale API Reference uses tags to categorize functions. The tags are `Community`, `Experimental`, `Toolkit`, and `Experimental (Toolkit)`. This section explains each tag.
```

Have to say, the interplay between Experimental and Toolkit and Experimental (Toolkit) is _super_ confusing.

File: api/api-tag-overview.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# API Reference Tag Overview
+
```
Comment (Loquacity) [inline api/api-tag-overview.md:2]:
```suggestion
# API Reference tag overview
```

File: api/api-tag-overview.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Timescale Documentation utilizes a tagging system to describe functions in the API Reference section. These tags include "Community", "Experimental", "Toolkit", and "Experimental" (for TimescaleDB Toolkit functions). Below is more information on what each tag represents.
+
+## Community <tag type="community">Community</tag>
+
```
Comment (Loquacity) [inline api/api-tag-overview.md:6]:
```suggestion
```

File: api/api-tag-overview.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+The community tag signifies that a TimescaleDB function is available under the Timescale Community Edition license, and not available under the Apache 2 Edition license. For more information, visit our [Timescale License comparison sheet][tsl-comparison].
```
Comment (Loquacity) [inline api/api-tag-overview.md:7]:
```suggestion
This tag indicates that the function is available under the Timescale Community Edition license, and are not available under the Apache 2 Edition license. For more information, visit our [Timescale License comparison sheet][tsl-comparison].
```

Where is the "Edition" coming in here? Shouldn't it just be "the Apache 2 license"?

File: api/api-tag-overview.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+## Experimental (TimescaleDB Experimental Schema) <tag type="experimental">Experimental</tag>
+
```
Comment (Loquacity) [inline api/api-tag-overview.md:10]:
```suggestion
```

File: api/api-tag-overview.md (hunk 2 above)
Comment (Loquacity) [inline api/api-tag-overview.md:2]:
```suggestion
```

File: api/api-tag-overview.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+The experimental tag denotes that a function is a part of the TimescaleDB experimental schema. This schema is not to be used in production, may include bugs and is subject to change in future versions. The objective of the schema is to allow for faster development of new features. The ultimate goal is to move experimental features and functions out of the experimental schema when they reach full maturity for normal production usage. [More information can be found on the Timescale blog][experimental-blog].
```
Comment (Loquacity) [inline api/api-tag-overview.md:11]:
```suggestion
This tag indicates that the function is included in the TimescaleDB experimental schema. Do not use experimental functions in production. Experimental features could include bugs, and are likely to change in future versions. The experimental schema is is used by Timescale to develop new features more quickly. If experimental functions are successful, they can move out of the experimental schema and go into production use. For more information about the experimental schema, [read the Timescale blog post][experimental-blog].
```

File: api/api-tag-overview.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+## Tookit <tag type="toolkit">Toolkit</tag>
```
Comment (Loquacity) [inline api/api-tag-overview.md:13]:
```suggestion
## Toolkit <tag type="toolkit">Toolkit</tag>
```

File: api/api-tag-overview.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (Loquacity) [inline api/api-tag-overview.md:14]:
```suggestion
```

File: api/api-tag-overview.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+The toolkit tag marks functions that belong to the TimescaleDB Toolkit extension. The functionality provided by the Toolkit is availble under the Timescale Community Edition license. For installation instructions, [visit our install guide][toolkit-install].
```
Comment (Loquacity) [inline api/api-tag-overview.md:15]:
```suggestion
This tag indicates that the function is included in the TimescaleDB Toolkit extension. Toolkit functions are available under the Timescale Community Edition license. For installation instructions, [see the installation guide][toolkit-install].
```

File: api/api-tag-overview.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+## Experimental (TimescaleDB Toolkit) <tag type="experimental-toolkit">Experimental</tag>
+
```
Comment (Loquacity) [inline api/api-tag-overview.md:18]:
```suggestion
```

File: api/api-tag-overview.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+The experimental tag, when used with the Toolkit tag, signfies that a function is still under active development. It may not handle corner cases or errors, and may have poor performance. Extion updates will drop database objects that depend on experimental features. This is particularly important for Timescale Cloud users, as the Toolkit extension is updated automatically. For more information, [read the TimescaleDB Toolkit docs][toolkit-docs].
```
Comment (Loquacity) [inline api/api-tag-overview.md:19]:
```suggestion
This tag is used with the Toolkit tag. It indicates a Toolkit function that is under active development. Do not use experimental toolkit functions in production. Experimental toolkit functions could include bugs, and are likely to change in future versions. These functions might not correctly handle unusual use cases or errors, and they could have poor performance. Updates to the TimescaleDB extension drop database objects that depend on experimental features like this function. If you use experimental toolkit functions on Timescale Cloud, this function is automatically dropped when the Toolkit extension is updated. For more information, [see the TimescaleDB Toolkit docs][toolkit-docs].
```

**Review by Loquacity — CHANGES_REQUESTED** [review]
Also, don't forget to flow/reflow the page.

File: api/api-tag-overview.md (hunk 4 above)
Comment (jacobprall) [reply api/api-tag-overview.md:7]: Matching it with the explanations here (prescribed by Mike and Prashant) https://docs.timescale.com/timescaledb/latest/timescaledb-license-comparison/

**Review by Loquacity — APPROVED** [review]
Just needs the lines reflowed.

**Pull request merged**